docs: Fix spelling of ressources to resources

janhoogeveen opened the pull request:
Contributes to documentation

Checklist:
- [X] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [X] checking not applicable
- [X] testing not applicable
- [X] screenshot not applicable

zoey-kaiser commented:
Hi @janhoogeveen 👋 

thank you for your contribution. I see that you also updated the formatting of the [NuxtAuthHanlder](https://github.com/sidebase/nuxt-auth/pull/251/files#diff-172107e289ab72242c60d324970fd81bc083a85917e8fc9e420b16c7e9729642). Could you please undo this change before we proceed?

Thank you!

janhoogeveen commented:
> Hi @janhoogeveen 👋
> 
> thank you for your contribution. I see that you also updated the formatting of the [NuxtAuthHandler](https://github.com/sidebase/nuxt-auth/pull/251/files#diff-172107e289ab72242c60d324970fd81bc083a85917e8fc9e420b16c7e9729642). Could you please undo this change before we proceed?
> 
> Thank you!

Woops. Changed it for you. Guess prettier got a bit too excited without my knowledge.

zoey-kaiser reviewed (approved):
Approved! Thanks for the PR @janhoogeveen. Before I merge I wanted to ask @BracketJohn about the redirects from the old URLs: 

By changing all these routes people who may use an older version of NuxtAuth may be directed to one of the outdated URLs. Do we take this risk into account or should we add some temporary redirects to the new links?

BracketJohn commented on docs/content/6.resources/1.external-blogs-guides-examples.md:
```diff
@@ -0,0 +1,10 @@
+-   [How to expand the user data object](https://github.com/sidebase/nuxt-auth/issues/61#issuecomment-1330946022)
+-   [Docus NuxtAuthHandler that shows how to manually fetch an external JWT token, how to expand the user data and how to track session expiry manually](https://gist.github.com/madsh93/b573b3d8f070e62eaebc5c53ae34e2cc)
+
+Note: If you're looking to make something work in `nuxt-auth` the nice thing is that you can use most `NextAuth.js` resources to help you along the way, as the `NuxtAuthHandler` behaves identical to the [`NextAuth` handler](https://next-auth.js.org/configuration/initialization).
```
this file was also auto-formatted (compare it to the old one below) - please revert this / make sure that you just copy paste it

janhoogeveen replied:
Fixed!

BracketJohn commented on docs/content/6.resources/3.security.md:
as above - don't do 2 spaces instead of 1 between the dash and the text please (:

janhoogeveen replied:
Also fixed!

BracketJohn reviewed (commented):
> Do we take this risk into account or should we add some temporary redirects to the new links?

let's make sure to add (temporary) redirects - it's hopefully failry simple with docus?

zoey-kaiser commented:
I will merge this now. I need to add the redirects in https://github.com/sidebase/docs. I will open a PR once I added them there!
